talker/command.py

```diff
@@ -199,7 +201,7 @@ def send(self):
 
     def on_sent(self):
         if not self.ack_timer:
-            self.ack_timer = Timer(expiration=TALKER_CONTEXT.ack_timeout)  # this is expiration on the agent ack'ing the command
```

Comment:
Is this context used anywhere in weka's code?

Reply:
Yes...